examples/dts/dts_activity_sequence.py

```diff
@@ -52,18 +51,18 @@ def sequence(ctx: task.OrchestrationContext, _):
 arm_scope = "https://durabletask.io/.default"
 token_manager = AccessTokenManager(scope = arm_scope)
 
-metaData: list[tuple[str, str]] = [
+meta_data: list[tuple[str, str]] = [
```

Comment:
The fact that we internally use the metadata dictionary to pass task hub information is an implementation detail that developers should not need to know or care about. This dictionary is intended to be used in cases where the developer needs to do something custom that isn't natively understood by the SDK. Task hubs do not fit into this bucket, because they are a first class concept in DTS. We should instead allow the developer to pass the task hub name directly in the DurableTaskSchedulerWorker and client class constructors in a type-safe way via a typed `taskhub: str` parameter that doesn't require them to look at documentation.